### PR TITLE
Updated SSE list of responsibilities

### PR DESCRIPTION
**Pull request by mirie**
This is pulled from: https://docs.google.com/spreadsheets/d/1c0Ags_6-aJEEBuMC46pw5hshsmOD20IKuhPjigBbcSM/edit#gid=1241753057

### CODE REVIEW

File: Positions/SeniorSoftwareEngineer.md [hunk 1]
```diff
@@ -1,36 +1,102 @@
-## What does "Senior Software Engineer" mean?
+## Senior Software Engineer (SSE)
 
-N.b. Not every single one of these criteria needs to be satisfied to be considered senior. But seniority comprises, in some combination or other, the majority of these.
+_Not every single one of these criteria needs to be satisfied to be considered a full JSE. But the role comprises, in some combination or other, the majority of these._
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: "a full JSE" -> "a full SSE"

File: Positions/SeniorSoftwareEngineer.md [hunk 2]
```diff
@@ -1,36 +1,102 @@
...
+- Able to effectively QB at least one area of the codebase
+- Familar/Comfortable with more than one area of the codebase
 
-### Automated testing
-- Facility with unit and end-to-end testing
-- Significant contributions to our test coverage
-- Could be more front-end (Wraith, QUnit) or back-end (Drupal unit/web tests, Behat)
+### Impact
+- Scopes and stages work into well-defined milestones to avoid a monolithic deliverable
+- Regularly delivers software on-time and is constantly working to make accurate estimates and deliver on those estimates
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: "on-time" -> "on time"

File: Positions/SeniorSoftwareEngineer.md [hunk 3]
```diff
@@ -1,36 +1,102 @@
...
+- Contributes to a framework or development practice. Has enough experience here to have a point of view, and can argue for systemic improvements
+- Possesses empathy with the user of the software they are producing and uses that empathy to guide decision-making
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: "decision-making" -> "decision making"

File: Positions/SeniorSoftwareEngineer.md [hunk 4]
```diff
@@ -1,36 +1,102 @@
...
+- Consistently writes code that sets the standard for quality and maintainability
+- Handles open-ended problems & ambiguity well
 
-### Architectural contributions
-- Major contributions to our JavaScript, Drupal, Laravel, Neue frameworks
-- Architecture-level consulting with other engineers
-- Communicate and solve problems with Product, Campaigns, Biz Dev, others
+### Communication
+- Clearly communicates timeline, scope and technical concerns
+- Makes others better through code reviews, documentation, technical guidance, and mentoring or serving as a Tech Lead (technical or team lead) on a project.
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: Remove period

File: Positions/SeniorSoftwareEngineer.md [hunk 5]
```diff
@@ -1,36 +1,102 @@
...
+- Regular consults with other developers on architecture, best practices, and documentation
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: "Regular" -> "Regularly"

File: Positions/SeniorSoftwareEngineer.md [hunk 6]
```diff
@@ -1,36 +1,102 @@
...
+- Proactively identifies problems with requirements (lack of clarity, inconsistencies, technical limitations) for their own and adjacent work, and communicates these issues early to help course-correct
 
-### FRONT-END: JavaScript requirements
-- Significant contributions to JS behaviors
-- Test coverage
+### Leadership
+- Has a mentor relationship with other engineers. This is broadly defined as serving as an effective listener, giving constructive critism often, and acting as a champion for others in order to facilitate growth. Knowledge sharing happens often via pairing, design review, code reviews, and presentations.
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: Remove period

File: Positions/SeniorSoftwareEngineer.md [hunk 7]
```diff
@@ -1,36 +1,102 @@
...
+
+### Additional Tech Lead Role Responsibilities
+There are additional responsibilities for an SSE who has an additional tech lead (technical and/or team) role. The [Technical Lead](Roles/TechLead.md) description provides further details on the overall responsibilities of this role. The following outlines responsibilities for the tech lead role using the same categories as above:
+
+### Technical Skill/Knowledge
+
+##### Technical
+- Deep knowledge and experience with the project(s) leading
+- Makes design decisions for the project(s) responsible for
+- Owns the technical testing and performance plan for their projects"
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: Remove double-quote at end of line

File: Positions/SeniorSoftwareEngineer.md [hunk 8]
```diff
@@ -1,36 +1,102 @@
...
+
+
+### Impact
+##### Technical & Team
+- Partners with Product to manage scope and deliverables for the technical side of the product roadmap.
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: Remove period

File: Positions/SeniorSoftwareEngineer.md [hunk 9]
```diff
@@ -1,36 +1,102 @@
...
+- Sets direction at the project/service level and consistently influences decision-making for the project/team
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: "decision-making" -> "decision making"

File: Positions/SeniorSoftwareEngineer.md [hunk 10]
```diff
@@ -1,36 +1,102 @@
...
+
+### Leadership
+
+##### Technical & Team
+- Participates in the Monthly Architecture Meeting; provides feedback on projects outside of their core area
+- Able to identify areas of strategic technical debt and provide cost/benefit analysis for eliminating this debt and suggested timelines for how to prioritize it
+
+##### Team
+- Comfortable managing team members with different skill sets and technical areas of focus
+- Identifies and proposes strategies around technical problems affecting their team, communicates standards and gets buy-in on solutions"
```
Comment (mshmsh5000) [inline Positions/SeniorSoftwareEngineer.md]: Remove double-quote

File: Positions/SeniorSoftwareEngineer.md (hunk 1 above)
Comment (mirie) [reply Positions/SeniorSoftwareEngineer.md]: Ah, d'oh. I actually want to keep the original line. Just without the "N.b"

File: Positions/SeniorSoftwareEngineer.md (hunk 3 above)
Comment (mirie) [reply Positions/SeniorSoftwareEngineer.md]: Updated both occurrences in the spreadsheet as well

**Comment by mirie** [comment]
Omg.... I can't deal with my inconsistent usage of periods and the scattering of double quotes that I _thought_ I removed!!!! 

long-stare

@mshmsh5000 updated with fixes.

**Pull request merged**